medium/__init__.py

```diff
@@ -7,84 +7,139 @@
 
 import requests
 
-BASE_PATH = "https://api.medium.com"
 
+def _request(method, path, access_token, json=None, form_data=None,
```

Comment:
Now that I'm looking at this, I don't think I actually like `_request` as a standalone function — it reduces some complexity by giving it access to less info, but by that argument we could split out all the methods. And it's nice for it to just have access to `access_token`, since it's something it _always_ needs but never changes from the caller's perspective.